### ISSUE TITLE
LDAP Strategy not authenticating users

### ISSUE THREAD

**Issue by Alchemi1963** [Requarks/wiki#4521]
### Actual behavior
Not authenticating users

### Expected behavior
Authenticating users

### Steps to reproduce the behavior
Configure LDAP, trying to log in.

So I'm trying to make Wiki.js work with our OpenLDAP implementation, but it doesn't appear to authenticate users; when loggin in I get: Invalid email / username or password.

I already enabled the LDAP Debug flag, but it didn't change anything in the console output...
I triple checked the login data, I verified the search filter directly in our OpenLDAP manager, the mail property also exists and can be requested (tested with ldapsearch). I feel like a tried everything at this point....

I have been at this for the whole day now and at a loss currently.

Does anyone have an idea of what this could be?

![image](https://user-images.githubusercontent.com/21000391/135494391-91540144-a355-4c9f-92f3-7176602ec21b.png)
![image](https://user-images.githubusercontent.com/21000391/135494433-b5fba213-31b2-4e2b-bf5d-fdeffb9ebf9d.png)

**Comment by Alchemi1963** [comment]
Nevermind, I just figured out the admin bind dn was wrong in two ways...
I would however, like to know why I didn't see anything in the console...

**Comment by elia427** [comment]
@Alchemi1963  Same issue here. How did you figure it out? Could you share the correct inputs to fields?

**Comment by mindcloud69** [comment]
@Alchemi1963 Can you please post your fix.

**Comment by Alchemi1963** [comment]
Sorry, I am not sure if my fix would work for you guys. But anyway:
In the admin bind dn I had: cn='admin',dc=bolkhuis.nl,dc=nl
However, the cn doesn't require de quotes and de dc isn't bolkhuis.nl; it's simply bolkhuis...

My advice for you guys, assuming you're still having the issue: triple check your settings, especially stuff like the bind dn.
I checked it like 5 times and I still didn't see it... I just had to look really carefully at all the details.
